Remotion/Web/ClientScript/Scripts/DropDownMenu.ts

```diff
@@ -865,6 +927,11 @@ class DropDownMenu
           anchor.focus();
         }
       }
+
+      // A selection change might trigger hover events that, in turn, would trigger selection events
+      // So we ignore mouse events for a small amount of time to deal with cascading events triggered by the selection change
+      DropDownMenu._ignoreMouseEvents = true;
+      setTimeout(() => DropDownMenu._ignoreMouseEvents = false, 250);
```

Comment:
can we shorten this to 50-100ms? I.e. do we just need to deblock the cpu or do we need to deblock the human's jittery hand?

Reply:
Firefox takes its time to report the relevant events that we want to block. A "high" timeout is not a problem here as we are only blocking hover effects, which means that the only bad thing would be that a user who does a keyboard selection cannot select something with the mouse in the next 250ms.

Reply:
please make this a cosntnat and add a comment regarding Firefox. And yes, users that might immeditately do a keyboard press and fall into the e.g. < 250ms is a real scenario in QA. (we have had some fun bug reports form this)